app/templates/request/new_request_agency.html

```diff
@@ -113,6 +118,27 @@ <h4 class="modal-title">Confirm Delete</h4>
                         </div>
                     </div>
                 </div>
+                <div class="modal fade" id="category-warning-modal" tabindex="-1" role="dialog">
+                    <div class="vertical-alignment-helper">
+                        <div class="modal-dialog vertical-align-center">
+                            <div class="modal-content">
+                                <div class="modal-header">
+                                    <h4 class="modal-title">Warning</h4>
+                                </div>
+                                <div id="category-warning-text" class="modal-body">Selecting this option will remove any
```

Comment:
Shouldn't we have this text defined in only one location? It's in the JS as well. Might cause confusion if we have to change it at a later date. 

Can we use the JS to fill in the div with the text specified in that file?

Applies to all occurrences of `#category-warning-text` and `#category-info-text`

Reply:
I'll keep all static text in the JS